pages/linux/getcap.md

```diff
@@ -0,0 +1,12 @@
+# getcap
+
+> Command to display the name and capabilities of each specified file.
+> More information: <https://www.man7.org/linux/man-pages/man8/getcap.8.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/getcap>.
```

Reply:
@saikatsgupta Can you commit this change?
 This PR is ready to be merged after this.